scripts/ci/build.sh

```diff
@@ -171,9 +171,7 @@ cat >>$testsrc_dir/setup.py <<EOL
                        '**/*.txt',
                        '**/*.txt',
                        '**/*.xml',
```

Comment:
Even though the document https://setuptools.pypa.io/en/latest/userguide/datafiles.html mentions:

> The `package_data` argument is a dictionary that maps from package names to lists of **glob patterns.**

`**` actually doesn't work: 
- https://github.com/pypa/setuptools/issues/1806